### PR TITLE
First pass at a fully documented README

### PR DESCRIPTION
**Pull request by lazypower**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,70 @@
...
+
+## Usage
+
+    juju deploy ~containers/bundle/logstash-core
+
+## Testing the deployment
+
+The services provide extended status reporting to indicate when they are ready:
+
+    uju status --format=tabular
```
Comment (johnsca) [inline README.md]: *j

**Comment by johnsca** [comment]
Other than the tiny typo in my previous comment, +1

File: README.md [hunk 2]
```diff
@@ -1,3 +1,70 @@
-# Logstash Core
+# Logstash Core - subject to renaming
 
-Basic service formation for standing up a logstash cluster
+The Elastic Stack — that's Elasticsearch, Logstash, Kibana, and Beats — are open
```
Comment (sparkiegeek) [inline README.md]: You mention Beats here but it's not mentioned in the text itself..

Comment (lazypower) [inline README.md]: You are correct! I suppose I should remove that until I have the Beat layer written.

File: README.md (hunk 1 above)
Comment (lazypower) [inline README.md]: Good catch :) Thanks

**Pull request merged**